emoji/src/main/java/com/vanniktech/emoji/EmojiPopup.java

```diff
@@ -149,15 +154,21 @@ public void toggle() {
       if (isKeyboardOpen) {
         // If the keyboard is visible, simply show the emoji popup.
         showAtBottom();
-      } else {
+      } else if (editInterface instanceof View) {
+        /* Uses properties unique to certain Android classes (those that inherit from
+          android.view.View */
+
         // Open the text keyboard first and immediately after that show the emoji popup.
-        emojiEditText.setFocusableInTouchMode(true);
-        emojiEditText.requestFocus();
+        ((View) editInterface).setFocusableInTouchMode(true);
```

Comment:
Can you describe in the interface that the one who's implementing this also needs to a `View`?